src/Models/CohortsModel.php

```diff
@@ -53,4 +53,10 @@ public function getAll(): array
 
         return $cohorts;
     }
+
+    public function getDate (){
+        $query = $this->db->prepare("SELECT `id`, `date` FROM `cohorts`;");
+        $query->execute();
+        return $query->fetchAll();
+    }
```

Comment:
This returns multiple dates so it should be `getDates` or `getCohortDates`.

But also why don't you juse use the `getAll` to get a list of all of the cohorts and then access the date from those entities? That's what I meant by saying it looked like this had already been done.